packages/eslint-plugin-stylable/src/stylable-es-lint.ts

```diff
@@ -20,11 +21,11 @@ export default createRule({
     defaultOptions: [{ exposeDiagnosticsReports: false, resolveOptions: {} }], // TODO: allow to pass resolve config
     create(context, options) {
         const [{ exposeDiagnosticsReports, resolveOptions }] = options as Options;
-        const moduleResolver = createDefaultResolver(nodeFs, resolveOptions);
+        const moduleResolver = createDefaultResolver(fs, resolveOptions);
 
         const stylable = Stylable.create({
-            fileSystem: nodeFs,
-            projectRoot: '/',
+            fileSystem: fs,
+            projectRoot: process.cwd(),
```

Comment:
this is the only behavior change. `/` is surely wrong.

Reply:
nice